.github/workflows/build.yaml

```
@@ -254,6 +262,41 @@ jobs:
           build-args: |
             JAR=edc-dataplane/${{ matrix.name }}/build/libs/${{ matrix.name }}.jar
           push: |
-            ${{ (needs.secret-presence.outputs.CXNG_GHCR_PAT && github.event_name != 'pull_request' && 'true') || 'false' }}
+            ${{ (github.event_name != 'pull_request' && 'true') || 'false' }}
           tags: ${{ steps.edc_dataplane_meta.outputs.tags }}
           labels: ${{ steps.edc_dataplane_meta.outputs.labels }}
+
+  publish-to-github-packages:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    needs: [secret-presence, verify-formatting, build-controlplane, build-dataplane, build-extensions]
+    if: |
+      needs.secret-presence.outputs.GPG_PASSPHRASE && needs.secret-presence.outputs.GPG_PRIVATE_KEY && github.event_name != 'pull_request'
+    steps:
+      # Set-Up
+      - name: Checkout
+        uses: actions/checkout@v3.3.0
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3.10.0
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+          cache: 'gradle'
+      - name: Import GPG Key
+        uses: crazy-max/ghaction-import-gpg@v1
+        env:
+          GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
+          PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+
+      # publish snapshots
+      - name: Publish snapshot versions
+        run: |-
+          echo "Publishing Version $(grep -e "defaultVersion" gradle.properties | cut -f2 -d"=") to Github Packages"
+          ./gradlew publishAllPublicationsToGitHubPackagesRepository
+        env:
+          REPO: ${{ github.repository }}
+          GITHUB_PACKAGE_USERNAME: ${{ github.actor }}
+          GITHUB_PACKAGE_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
```

Missing empty line at EOF
```suggestion
          GITHUB_PACKAGE_PASSWORD: ${{ secrets.GITHUB_TOKEN }}

```